tests/ui/pages/desktop/regions/sorter.py

```diff
@@ -0,0 +1,21 @@
+from selenium.webdriver.common.by import By
+
+from pages.desktop.base import Base
+
+
+class Sorter(Base):
+    """Helper class for sorting."""
+    _root_locator = (By.ID, 'sorter')
+    _sort_by_type_locator = (By.CSS_SELECTOR, 'ul > li')
+    _updating_locator = (By.CSS_SELECTOR, '.updating')
+
+    def sort_by(self, sort):
+        """Clicks the sort by type requested."""
+        elements = self.find_elements(*self._sort_by_type_locator)
+        # Look for requested sort type and if found, click it and return
+        for el in elements:
+            if sort == el.text:
+                el.click()
+                self.wait.until(lambda _: not self.is_element_present(
+                    *self._updating_locator))
```

Comment:
Also using geckodriver 16 seems to wait for that updating element to disappear before proceeding.